src/praisonai/praisonai/cli/features/onboard.py

```
@@ -580,7 +611,25 @@ def _run_plain(self) -> None:
+
+        # Parity with rich flow: install daemon by default (no prompt)
+        # when every selected platform has a token captured.
+        if self.selected_platforms and all(p in self.tokens for p in self.selected_platforms):
+            try:
+                from praisonai.daemon import install_daemon
+                result = install_daemon(config_path=str(cfg_path))
+                if result.get("ok"):
+                    print(f"✓ {result.get('message', 'Daemon installed')}")
+                else:
+                    print(f"✗ {result.get('error', 'Daemon install failed')}")
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)